mobile/function/map_page_function.js

```diff
@@ -32,11 +30,12 @@ const GetMapPage = (routeId, hasLocation) => {
 
         const json = await response.json();
         setData(json);
+        console.log(json);
```

Comment:
ook console.log weg halen tenzij je die gebruikt die er met een reden is natuurlijk